src/mixins/raster-layer-point-mixin.js

```diff
@@ -96,11 +96,37 @@ function getColor(color, layerName) {
   }
 }
 
+function isValidPostFilter(postFilter) {
+  const {
+    operator,
+    min,
+    max,
+    aggType,
+    value,
+    table,
+    custom
+  } = postFilter
+
+  if (value && (aggType || custom)) {
+    if ((operator === "not between" || operator === "between") && (typeof min === "number" && !isNaN(min)) && (typeof max === "number" && !isNaN(max))) {
+      return true
+    } else if ((operator === "equals" || operator === "greater than") && (typeof min === "number" && !isNaN(min))) {
+      return true
+    } else if (operator === "greater than" && typeof min === "number" && !isNaN(min)) {
```

Comment:
Isn't this case covered by the "else if" above?

Reply:
@bmatcuk, yes it is. Good catch. Thank you.